boards/avsextrem/board_init.c

```diff
@@ -41,19 +41,19 @@ void init_clks1(void)
     PLLCON &= ~0x0002;
     pllfeed();
 
-    while (PLLSTAT & BIT25);  /* wait until PLL is disconnected before
+    while (PLLSTAT & BIT25) {}/* wait until PLL is disconnected before
```

Comment:
why not moving the comment (here and below) **above** the loop statement as discussed f2f?

Otherwise, nice job 👍

Reply:
I will move it in another PR, because there are files that are not being touched by this one :) (I already have the script for doing so)